pkg/config/config_params.go

```diff
@@ -102,8 +102,10 @@ type Config struct {
 	LogSeverityScreen string `config:"oneof(DEBUG,INFO,WARNING,ERROR,CRITICAL);INFO"`
 	LogSeveritySys    string `config:"oneof(DEBUG,INFO,WARNING,ERROR,CRITICAL);INFO"`
 
-	HealthEnabled                   bool `config:"bool;false"`
-	HealthPort                      int  `config:"int(0,65535);9098"`
+	HealthEnabled                   bool   `config:"bool;false"`
+	HealthHost                      string `config:"string;"`
```

Comment:
Could you default this to localhost, like this: `config:"string;localhost"`

Reply:
Done!